zip-0032.rst

```diff
@@ -309,6 +314,71 @@ Sprout child key derivation
 - Use :math:`I_R` as the child chain code :math:`\mathsf{c}_i`.
 
 
+Specification: Orchard key derivation
+=====================================
+
+The derivation mechanism for Sapling addresses specified above incurs significant complexity to support
+non-hardened derivation. In the several years since Sapling was deployed, we have seen no use cases for
+non-hardened derivation appear. With that in mind, we define Orchard key derivation very similarly to
+Sprout above: only hardened derivation is supported.
+
+Orchard extended keys
+---------------------
+
+We represent an Orchard extended spending key as :math:`(\mathsf{sk, c}),` where :math:`\mathsf{sk}`
+is the normal Orchard spending key (opaque 32 bytes), and :math:`\mathsf{c}` is the chain code.
```

Comment:
Chain code is not defined in the terminology of this document yet, could you add it?